### DISCUSSION TITLE
Вопрос про Beta и Edge

### DISCUSSION THREAD

**Discussion by Eyeborgs** [AdguardTeam/AdGuardHome#3688] [Q&A]
Почему в Beta\Edge отключен Disallowed clients и будет он включен в Release версии?

**Comment by ainar-g** [comment]
Добрый вечер.  Мы ничего не отключали.  Возможно, у вас список разрешённых клиентов заполнен?  В таком случае заблокированные клиенты, очевидно, игнорируются, так как заблокированы все, кто не разрешён.  С недавних пор мы делаем это более очевидным.

**Reply by Eyeborgs** [reply]
Beta
![image](https://user-images.githubusercontent.com/46345396/135498341-39bb4f47-9f0e-41af-a510-2e7be38ec2fc.png)

Edge 
![Безымянный](https://user-images.githubusercontent.com/46345396/135498576-d792657e-2fb7-481d-b86b-be5079952aa6.png)

**Reply by ainar-g** [reply]
Это именно то, о чём я писал.  У вас заполнено поле «Разрешённые клиенты».  Все клиенты кроме тех, кто в этом списке, заблокированы итак.

**Reply by Eyeborgs** [reply]
@ainar-g спасибо я понял, а то в версии 0.106.3 не так 
но все же мне нравилось как раньше было там где и «Разрешённые клиенты» и «Запрещённые клиенты»

**Reply by ainar-g** [reply]
Вы имеете в виду возможность *редактировать* поле «Запрещённые клиенты»?  Над этим можем подумать, но сделано это было как раз для того, чтобы подчеркнуть, что оно игнорируется, когда другое заполнено.

**Reply by Eyeborgs** [reply]
я считаю лучше оставить как есть с версии 0.106.3, и так все раньше было понятно 

Или сделать какую либо инструкцию почему отключено..

**Reply by Eyeborgs** [reply]
потому что после обновления с версии 0.106.3 на 0.107.* может у многих возникнуть вопросы как у меня..

**Reply by ainar-g** [reply]
Но объяснение уже там.

 >  Если настроены разрешённые клиенты, это поле игнорируется.

**Reply by Eyeborgs** [reply]
@ainar-g хорошо, возможно я не заметил 
спасибо что ответили на мой вопрос
